internal/client/datadog/formatter_test.go

```diff
@@ -78,7 +80,7 @@ var _ = Describe("Formatter", func() {
 		Expect(string(helper.Decompress(result[0]))).To(ContainSubstring(`"points":[[0,9.000000],[0,1.000000]]`))
 	})
 
-	It("properly assigns all values to split results when splitting", func() {
+	It("properly split metrics into two maps", func() {
 		// first test a scenario where we're not splitting as there's just one point
 		m := make(map[metric.MetricKey]metric.MetricValue)
```

Comment:
This part is outdated too. We should always be splitting as soon as the resulting payload is above the limit